src/main/java/org/spdx/sbom/gradle/SpdxSbomPlugin.java

```diff
@@ -128,55 +110,33 @@ private void createTaskForTarget(
                             .getResolutionResult()
                             .getRootComponent();
 
-                    Configuration pomsConfig =
-                        project
-                            .getConfigurations()
-                            .detachedConfiguration(
-                                project
-                                    .getConfigurations()
-                                    .getByName(configurationName)
-                                    .getIncoming()
-                                    .getResolutionResult()
-                                    .getAllComponents()
-                                    .stream()
-                                    .filter(rcr -> rcr.getId() instanceof ModuleComponentIdentifier)
-                                    .map(rcr -> rcr.getId().getDisplayName() + "@pom")
-                                    .map(pom -> project.getDependencies().create(pom))
-                                    .toArray(Dependency[]::new));
-                    t.getPoms()
-                        .putAll(PomResolver.newPomResolver(project).effectivePoms(pomsConfig));
```

Comment:
This was causing resolution issues in my project because of variants and mismatched attributes.

Reply:
You can see the information Gradle uses to resolve the variant here - https://repo1.maven.org/maven2/com/eygraber/android-date-time-input-common/0.9.16/android-date-time-input-common-0.9.16.module